spec/support/features/session_helper.rb

```diff
@@ -50,6 +50,7 @@ def sign_in_user(user = create(:user))
     end
 
     def sign_in_before_2fa(user = create(:user))
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
Good question. I could see it go both ways. It's only used by feature specs, so in only 4 files. The controller specs use a method with the same name in `controller_helper.rb`. None of those tests rely on the `prefill_otp_codes?` feature to be `false`, but a few of them rely on it being `true`, so I think it would make it easier to write a test that needs the code to be prefilled since you wouldn't have to write that `allow` line for each test.

Most of those tests simply want a user to end up on the page that prompts them to enter the OTP